UI/installer/mp-installer.nsi

```diff
@@ -42,18 +54,14 @@ RequestExecutionLevel admin
 !define MUI_FINISHPAGE_RUN_TEXT "Launch OBS Studio ${SHORTVERSION}"
 !define MUI_FINISHPAGE_RUN_FUNCTION "LaunchOBS"
 
-; GPL is not an EULA, no need to agree to it.
```

Comment:
This should be kept. The "live" installer probably was not updated to include this.

Reply:
The current installer still shows the license, the major difference is that the bottom text saying "You are now aware of your rights." was removed. Which part specifically do you want reincluded? The comment in the installer script, or the text itself?

Reply:
Really it's more the button text that's the biggest issue. The button should say "Next" instead of the default "I agree".

Reply:
Yes, this is already handled in the new script, on L60.